waargonaut.cabal

```diff
@@ -45,13 +45,8 @@ extra-source-files:
 homepage:           https://github.com/haskell-waargonaut/waargonaut
 bug-reports:        https://github.com/haskell-waargonaut/waargonaut/issues
 tested-with:
-  GHC ==8.0.2 || ==8.2.2 || ==8.4.4 || ==8.6.5 || ==8.8.4 || ==8.10.3
+  GHC ==8.4.4 || ==8.6.5 || ==8.8.4 || ==8.10.7 || ==9.0.1
 
-custom-setup
-  setup-depends:
-      base           >=4.9 && <5
-    , Cabal
-    , cabal-doctest  >=1   && <1.1
```

Comment:
General practice discourages this custom-setup clause. We should use `cabal-docspec` instead.